acquamanCommon.pri

```diff
@@ -844,7 +845,8 @@ SOURCES += \
 	source/beamline/AMExclusiveStatesEnumeratedControl.cpp \
     source/ui/beamline/AMControlStopButton.cpp \
     source/ui/beamline/AMControlToolButton.cpp \
-    $$PWD/source/ui/AMToolButton.cpp
+    $$PWD/source/ui/AMToolButton.cpp \
```

Comment:
I believe this is a factor of different Qt versions. 
When I add a class it appears in the project file in the format
```
source/blah/thing
```
whereas it seems as if when either Jessie or Darren add it it prepends the $$PWD onto the front. I have seen no observable difference between both. If the $$PWD prefix is removed everything still works fine.